src/commands/test/slightfile.toml

```diff
@@ -0,0 +1,5 @@
+specversion = "0.2"
+
+[[capability]]
```

Comment:
@danbugs, I thought it was interesting that if I didn't include a capability, slight would fail when I ran `handle_run`. Is that a bug? Must one use at least 1 capability?

Reply:
File an issue for this please!

Reply:
Added.